composer.json

```
@@ -51,6 +51,7 @@
 		"friendsofphp/php-cs-fixer": "^3.60.0",
 		"helmich/typo3-typoscript-lint": "^3.1.1",
 		"php-coveralls/php-coveralls": "^2.7.0",
+		"phpcompatibility/php-compatibility": "^9.3",
```

For development dependencies, please always use the exact patch-level version (as we don't risk breaking anything with this):

```suggestion
		"phpcompatibility/php-compatibility": "^9.3.5",
```